### PR TITLE
Add contributor

### PR DESCRIPTION
**Pull request by VladimirAlexiev**
Add myself as contributor.
- I've added tons of issues and suggestions
- And now made two PRs: https://github.com/w3c-ccg/traceability-vocab/pull/938, https://github.com/w3c-ccg/traceability-vocab/pull/937

### CODE REVIEW

File: docs/openapi/components/schemas/common/CBPEntry.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ $linkedData:
   term: CBPEntry
   '@id': https://w3id.org/traceability#CBPEntry
 title: CBP Entry
-description: CBP Form 3461 for Entry/Immediate Delivery (https://www.cbp.gov/sites/default/files/assets/documents/2016-Jun/CBP%20Form%203461%20-%20ACE%20Fillable.pdf and https://www.cbp.gov/sites/default/files/assets/documents/2016-Jun/CBP%203461%20Instructions_0.pdf).
+description: CBP Form 3461 for Entry/Immediate Delivery ([Fillable Form](https://www.cbp.gov/sites/default/files/assets/documents/2016-Jun/CBP%20Form%203461%20-%20ACE%20Fillable.pdf) and [Instructions](https://www.cbp.gov/sites/default/files/assets/documents/2016-Jun/CBP%203461%20Instructions_0.pdf)).
```
Comment (TallTed) [inline docs/openapi/components/schemas/common/CBPEntry.yml:5]: overlaps with #937

File: docs/openapi/components/schemas/credentials/CBP3461EntryCredential.yml [hunk 2]
```diff
@@ -8,7 +8,7 @@ tags:
   - Agriculture
   - Oil and Gas
   - Other
-description: Customs and Border Protection Form 3461 Immediate Delivery Certification (https://www.cbp.gov/sites/default/files/assets/documents/2016-Jun/CBP%20Form%203461%20-%20ACE%20Fillable.pdf and https://www.cbp.gov/sites/default/files/assets/documents/2016-Jun/CBP%203461%20Instructions_0.pdf).
+description: Customs and Border Protection Form 3461 Immediate Delivery Certification  ([Fillable Form](https://www.cbp.gov/sites/default/files/assets/documents/2016-Jun/CBP%20Form%203461%20-%20ACE%20Fillable.pdf) and [Instructions](https://www.cbp.gov/sites/default/files/assets/documents/2016-Jun/CBP%203461%20Instructions_0.pdf)).
```
Comment (TallTed) [inline docs/openapi/components/schemas/credentials/CBP3461EntryCredential.yml:11]: overlaps with #937

**Review by mkhraisha — APPROVED** [review]

**Comment by nissimsan** [comment]
@VladimirAlexiev , any chance you can split this apart into cleaner PRs, pls? It's a lot of different stuff going on at once here.

**Comment by VladimirAlexiev** [comment]
Hi @nissimsan! This PR mixes:
- https://github.com/w3c-ccg/traceability-vocab/pull/937 (which is already merged)
- adding myself as contributor (which is the purported purpose of this PR)
- removing trailing spaces from each line (which is a good practice). I haven't even noticed that my Emacs made this change, sorry!

I have checked and it doesn't have anything more than this.

In the future I'll make sure each PR does one thing. Sorry!

**Review by mkhraisha — APPROVED** [review]

**Pull request merged**